etl/collections/multidim.py

```diff
@@ -16,16 +14,23 @@
 import pandas as pd
 import yaml
 from deprecated import deprecated
-from owid.catalog import Dataset, Table
+from owid.catalog import Table
 from sqlalchemy.engine import Engine
 from structlog import get_logger
 
 from apps.chart_sync.admin_api import AdminAPI
+from etl.collections.utils import (
```

Comment:
Nit - we're importing from the library itself, so using `from .utils import ...` could work, too.